uniswap/uniswap.py

```diff
@@ -401,6 +402,7 @@ def make_trade(
         fee: int = None,
         slippage: float = None,
         fee_on_transfer: bool = False,
+        route: Optional[List[AddressLike]] = None,
```

Comment:
Might lead to unexpected behavior if `route` is specified but not handled (such as in the eth_to_token and token_to_eth cases).